zerver/models.py

```diff
@@ -1559,6 +1559,17 @@ class UserBaseSettings(models.Model):
     ]
     user_list_style = models.PositiveSmallIntegerField(default=USER_LIST_STYLE_WITH_STATUS)
 
+    # Show unread counts for
+    UNREAD_COUNT_ALL_STREAMS = 1
+    UNREAD_COUNT_UNMUTED_STREAMS = 2
+    UNREAD_COUNT_NO_STREAMS = 3
+    UNREAD_COUNT_CHOICES = [
+        UNREAD_COUNT_ALL_STREAMS,
+        UNREAD_COUNT_UNMUTED_STREAMS,
+        UNREAD_COUNT_NO_STREAMS,
+    ]
+    show_unread_counts = models.PositiveSmallIntegerField(default=UNREAD_COUNT_UNMUTED_STREAMS)
```

Comment:
I think maybe `stream_unreads_display` would be a better name for this; I'd expect `show_unread_counts` to be a boolean, not an enum.

I think we've also been moving in the direction of naming the all-caps variables things like `STREAM_UNREADS_DISPLAY_ALL_STREAMS` -- since that helpfully puts these bundles of values into a bit of a namespace. You can see that with `USER_LIST_STYLE_COMPACT`, just above.